### PR TITLE
11.0 mig l10n it ddt

### PR DESCRIPTION
**Pull request by SilvioGregorini**
@SimoRubi here is my PR with all the work I've done so far.
I think the last two commits may be merged or squashed into a single one, yet I wasn't sure about it... so I left them as they were.

### CODE REVIEW

File: l10n_it_ddt/models/stock_picking_package_preparation.py [hunk 1]
```diff
@@ -628,19 +625,19 @@ def invoice_line_create(self, invoice_id, qty):
...
-                    else:
-                        res[quant.lot_id] += quant.qty
-        for lot in res:
-            if lot.product_id.tracking == 'lot':
-                res[lot] = formatLang(self.env, res[lot])
-            else:
-                # If not tracking by lots, quantity is not relevant
-                res[lot] = False
-        return res
+    # def quantity_by_lot(self):
```
Comment (SimoRubi) [inline l10n_it_ddt/models/stock_picking_package_preparation.py:628]: Either keep the method, or delete it

File: l10n_it_ddt/wizard/add_picking_to_ddt.xml [hunk 2]
```diff
@@ -21,12 +21,14 @@
...
             <field name="type">ir.actions.act_window</field>
             <field name="res_model">add.pickings.to.ddt</field>
+            <field name="binding_model_id" ref="stock.model_stock_picking"/>
             <field name="view_type">form</field>
             <field name="view_mode">form</field>
             <field name="view_id" ref="view_add_pickings_to_ddt"/>
             <field name="target">new</field>
         </record>
 
+    <!--
```
Comment (SimoRubi) [inline l10n_it_ddt/wizard/add_picking_to_ddt.xml:31]: Either keep the record, or delete it

**Review by SimoRubi — CHANGES_REQUESTED** [review]
Hi, thanks for your work.
Before adding new functionalities, I'd prefer to be sure that the migration commit works correctly so please keep the other 4 commits in another branch and let's work only on the migration.
You will be able to do another PR for your new functionalities directly in https://github.com/OCA/l10n-italy when the module is up and running in v11.

Moreover, I suggest not to comment the old things but directly remove them, they only confuse the reader.

In order to check if the module migrated to v11 works, we first have to migrate its dependencies, so please have a look at https://github.com/OCA/stock-logistics-workflow/pull/448

**Comment by SilvioGregorini** [comment]
Hi @SimoRubi, I'm splitting the branch right now.
I'm creating two new branches (one for the migration only, the other one for new features to be implemented). They should be ready this afternoon.

**Comment by SimoRubi** [comment]
Thanks, and please remember to have a look at https://github.com/OCA/stock-logistics-workflow/pull/448 when you have time, otherwise we can't
complete the migration of l10n_it_ddt

**Comment by SilvioGregorini** [comment]
@SimoRubi I think you might wanna cancel this PR.
PR #2 suffices for the migration commit, while the other commits will be left in my own branch while I develop them.

**Comment by SimoRubi** [comment]
Superseded by #2

**Pull request closed without merging**